docs/argo-server-sso.md

```diff
@@ -194,3 +194,21 @@ sso:
 # assuming customClaimGroupName: argo_groups
 workflows.argoproj.io/rbac-rule: "'argo_admins' in groups"
 ```
+
+## Filtering groups
+
+> v3.5 and above
+
+Configure the `filterGroupsRegex` field to filter the groups returned by the OIDC provider. Some use-cases for this include:
```

Comment:
```suggestion
You can configure `filterGroupsRegex` to filter the groups returned by the OIDC provider. Some use-cases for this include:
```

- this is optional, so use "you can", which is consistent with the rest of the docs
- the rest of the docs refer directly to the fields as nouns, without using "the X field"